### ISSUE TITLE
Godot Editor keeps closing by itself.

### ISSUE THREAD

**Issue by LucasBremm** [godotengine/godot#97994]
### Tested versions

- Reproducible on 4.3, 4.2 and 4.1

### System information

Godot v4.3.stable - Windows 10.0.22631 - Vulkan (Forward+) - dedicated NVIDIA GeForce RTX 3050 6GB Laptop GPU (NVIDIA; 32.0.15.6109) - 12th Gen Intel(R) Core(TM) i5-12450H (12 Threads)

### Issue description

I am developing a game in godot for some time now, and while I work at it, it tends to close on itself a lot, just working normally, then freezes for a sec and closes. Tried to run with console and verbose to see what could be happening, all I got was:

```Vulkan 1.3.280 - Forward+ - Using Device #0: NVIDIA - NVIDIA GeForce RTX 3050 6GB Laptop GPU```

On console every time it crashes, and on verbose:

```
glTF: accessor offset 0 view offset: 5356 total buffer len: 24684 view len 1104
glTF: Total meshes: 1
glTF: Creating mesh for: bote
Using present mode: Enabled
```

Verbose I tried only once. Appears similar to [#81670](https://github.com/godotengine/godot/issues/81670). But I am not sure and I don't have the same technical knowledge about Vulkan to retrace the steps easily.

### Steps to reproduce

Using the MRP I've linked together, just double click plate.glb, it will open the 3D viewer window, just messing around with the 3D model (clicking and dragging making it spin and roll) will make the editor freeze and close in a couple of seconds.

### Minimal reproduction project (MRP)

[MRP.zip](https://github.com/user-attachments/files/17300212/MRP.zip)

**Comment by LucasBremm** [comment]
Update: While re-reading the issue I cited above, I read a part that says that opening godot with the parameter "--single-window" might solve the issue, and for me looks like it does solve. Godot now runs a little slower, but no more problems with auto closing.

**Comment by SourceOfHTML** [comment]
I am unable to reproduce it on v4.3 or current (v4.4.dev.custom_build [04692d83c]) version. I went crazy with it, and even zoomed in and out a bunch.

The only difference I can pick out is that I have an AMD Graphics Card (RX580), whereas you seem to have an NVidia one. Or perhaps I'm not spinning the plate.glb _**vigorously**_ enough.

**Comment by LucasBremm** [comment]
> Or perhaps I'm not spinning the plate.glb _**vigorously**_ enough.

Nah, I don't think thats the case, I was not spinning so violently when the editor crashed.

Again citing the issue I think I have the same problem as ([#81670](https://github.com/godotengine/godot/issues/81670)), I believe it have to do with some software that is using vulkan and messing with it in a way to disturb godot. But again, I'm not at all proeficient at vulkan or graphics programming, so it can be anything...

**Comment by hakro** [comment]
I have not been able to reproduce this, even in multi-window mode, in 4.4.1
My specs Godot v4.4.1.stable - TUXEDO OS 24.04.1 LTS on Wayland - X11 display driver, Multi-window, 1 monitor - Vulkan (Forward+) - dedicated NVIDIA GeForce RTX 4060 Laptop GPU - 13th Gen Intel(R) Core(TM) i7-13700H (10 threads)

@LucasBremm do you still have the problem with the latest version without enabling single-window ?

**Comment by LucasBremm** [comment]
I'm really sorry for the time it took me to come back here. Following what was told on the issue ([#81670](https://github.com/godotengine/godot/issues/81670)) I uninstalled MSI Afterburner, I software I used to put an overlay on games to see informations like cpu usage, temperature, fps, etc etc. And it looks like solved the issue. Apparently that software messes with Vulkan and caused conflict with godot.
